Source/Pawnmorphs/Esoteria/PawnmorphPatches.cs

```diff
@@ -237,6 +237,13 @@ private static void MassPatchFormerHumanChecks([NotNull] Harmony harmonyInstance
 
             List<MethodInfoSt> methodsToPatch = new List<MethodInfoSt>();
 
+            // Patch doors expanded mod if present.
+            if (LoadedModManager.RunningMods.Any(x => x.PackageId == "jecrell.doorsexpanded"))
+            {
+                methodsToPatch.Add(AccessTools.Method("DoorsExpanded.Building_DoorExpanded:PawnCanOpen"));
```

Comment:
I'd add some checks to this if the method isn't found even if the mod is present, I figure a mod is going to have a more volatile API then the base game so some warning if something changes would be good

Reply:
That is a very good point. Though I think we have null checks later in the code before patching?

That is what triggers the null method warning

Reply:
Yeah, we have a null check line 314

Reply:
I mean a check at 243. if AccessTools returns a null it won't break anything because of the later check but we won't know the Doors Expanded patch isn't working anymore

Reply:
Right, so you want to be able to tell independently from core methods if a mod integration patch failed.